packages/react-native-reanimated/src/specs/ReanimatedViewProvider.ts

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+import type { HostComponent } from 'react-native';
+
+export default {} as HostComponent<any>;
```

Comment:
What does this file do?

Reply:
I missed a comment. This is a workaround for Next.js 😕. Next.js can’t resolve the `react-native/Libraries/Utilities/codegenNativeComponent` import, and during SSG, we can’t detect if it is a web environment, so we can't return a mock implementation. To avoid this issue, we need two versions of `ReanimatedViewProvide`: one with a `.ts` extension for Next.js and web, and another with a `.native.ts` extension for React Native (since only Metro understands platform-dependent extensions).